canonical-websites/tutorials.ubuntu.com
# Update run command with automagic demos

Update the run command to the latest version.

This should enable demos to build on this very PR!

# QA

A demo with more than one tutorial in it should appear in this very PR!

Also, please run `./run serve ./tutorials/*/` locally to make sure it indeed works as expected..
